OJS.Workers.SubmissionProcessors/Formatters/CheckerFormatterService.cs

```diff
@@ -0,0 +1,24 @@
+﻿namespace OJS.Workers.SubmissionProcessors.Formatters
+{
+    using System.Collections.Generic;
+
+    using OJS.Workers.Common.Extensions;
+
+    public class CheckerFormatterService
+        : ICheckerFormatterService
+    {
+        private readonly IDictionary<string, string> map;
+
+        public CheckerFormatterService()
+            => this.map = new Dictionary<string, string>()
+            {
+                { "trim-checker", "trim" },
+                { "c-sharp-code-checker", "csharp-code" },
+            };
+
+        public string Format(string obj)
+            => this.map.ContainsKey(obj.ToHyphenSeparatedWords())
+                ? this.map[obj.ToHyphenSeparatedWords()]
+                : obj.ToHyphenSeparatedWords();

```

Comment:
Why `ToHyphenSeparatedWords` instead of just using the name of the class, preventing unwanted behavior if renamed?

Reply:
Because we have dependencies to this name in the interactive engine. Change here will result in refactoring in three places

Reply:
I think the checker is called just "trim", not "trim-checker"